tests/SemanticVersionTest.php

```diff
@@ -47,32 +47,31 @@ public function testCanParseVersionWithPrereleaseAndBuild()
         $this->assertEquals('build2.4', $sv->build);
     }
 
+    /**
+     * @expectedException \InvalidArgumentException
+     * @expectedExceptionMessage not a valid semantic version
+     */
     public function testLeadingZeroNotAllowedInMajor()
     {
-        try {
-            SemanticVersion::parse('02.3.4');
-            $this->assertTrue(false, 'expected exception');
-            // can't use expectException method because we must support older PHPUnit
```

Comment:
In regards to this comment, what is the minimum PHPUnit version for your changes?

Reply:
I'll run our usual suite of tests against all of our supported PHP versions to be sure.